Casks/filebot.rb

```diff
@@ -3,11 +3,15 @@
   sha256 "15c92853503f17958e7fbbe5d4640458021327126a4b4c9410a9d3d8086e02dc"
 
   url "https://get.filebot.net/filebot/FileBot_#{version}/FileBot_#{version}.app.tar.xz"
-  appcast "https://app.filebot.net/update.xml"
   name "FileBot"
   desc "Tool for organizing and renaming movies, TV shows, anime or music"
   homepage "https://www.filebot.net/"
 
+  livecheck do
+    url "https://app.filebot.net/update.xml"
+    regex(/FileBot (\d(?:\.\d+)*)/i)
```

Comment:
```suggestion
    url "https://www.filebot.net/download.html"
    regex(/href=.*?FileBot[._-]v?(\d+(?:\.\d+)+)\.app\.t/i)
```

The XML file only links to the first-party download page, rather than the archive file used in the cask `url`. The first-party download page links to the `app.tar.xz` file, so it would be better to check that.